src/modules/enginesyncer.js

```diff
@@ -87,8 +88,19 @@ export default class EngineSyncer extends EventEmitter {
     this.commands = []
     this.treePosition = null
 
-    this.controller = new Controller(resolve(path), [...argvsplit(args)], {
-      cwd: dirname(resolve(path))
+    let fileExistsSync = path => {
+      try {
+        fs.accessSync(path, fs.F_OK)
+        return true
+      } catch {
+        return false
+      }
+    }
```

Comment:
Why not simply use `fs.existsSync()`?

Reply:
My understanding is that fs.existsSync is deprecated, but that information was outdated. I modified it to use fs.existsSync.